src/pages/studyView/StudyViewPageStore.ts

```diff
@@ -2397,6 +2470,18 @@ export class StudyViewPageStore {
             };
         }
 
+        if (!_.isEmpty(this.mutationProfiles.result)) {
```

Comment:
just want to double check that this is the right condition for including fusion table?  (mutationProfiles not empty which is used by MUTATED_GENES_TABLE as well)